src/BluetoothHciSocket.cpp

```diff
@@ -184,6 +187,17 @@ void BluetoothHciSocket::bindControl() {
   bind(this->_socket, (struct sockaddr *) &a, sizeof(a));
 }
 
+void BluetoothHciSocket::bindUser() {
+  struct sockaddr_hci a;
+
+  memset(&a, 0, sizeof(a));
+  a.hci_family = AF_BLUETOOTH;
+  a.hci_dev = 0;
```

Comment:
True, the hci_dev should be passed as parameter. I'll eventually fix this PR if my integration in noble is going well... (still struggling on SMP Confirm...)

Regarding the default, the difference with `bindRaw`, is that the device CANNOT be up in this mode (no bluetooth service, Cf README.md), so it would just be the first device...